modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -482,6 +490,91 @@ import {TextInput} from '@workday/canvas-kit-react/text-input';
 
 🤖 The codemod will handle the change of `hasError` to `error` for you automatically.
 
+### Icons
+
+All icon components, `Svg`, `SystemIcon`, `SystemIconCircle`, `AccentIcon`, `AppletIcon`, `Graphic`,
+now use
+[Canvas Tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs) and
+our
+[new styling utilities](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api).
+The component now supports the `cs` prop, but otherwise the API has not changed. It should behave
+identically as it did in previous versions.
```

Comment:
i guess since we removed the styles prop, this isn't exactly true, is this a breaking change?